hera_cal/io.py

```diff
@@ -1109,19 +1110,23 @@ def empty_arrays(self):
         self.nsample_array = (np.zeros_like(self.nsample_array) if self.nsample_array is not None else None)
 
 
-def read_hera_hdf5(filenames, bls=None, pols=None, full_read_thresh=0.002,
-                   read_data=True, read_flags=False, read_nsamples=False,
-                   check=False, dtype=np.complex128, verbose=False):
+def read_hera_hdf5(
+    filenames, bls=None, pols=None, keep_times: list[float] | None=None, full_read_thresh=0.002,
```

Comment:
I'm not familiar with this style of type hinting. Why is it`| None=None`?

Reply:
The following None is the default for the parameter. I agree it could look a litle silly, though I'm not sure how it would look best.